criu/sk-unix.c

```diff
@@ -2195,6 +2185,7 @@ static int add_fake_queuer(struct unix_sk_info *ui)
 	peer->ue->ino = FAKE_INO;
 	file_desc_add(&peer->d, peer_ue->id, &unix_desc_ops);
 	list_del_init(&peer->d.fake_master_list);
+	add_socket_table_entry(FAKE_INO, (struct socket_table_entry *)peer);
```

Comment:
I think it doesn't make sense to add fake sockets to sockets hashtable, because all fake sockets have the same inode number (0) and we use fake sockets only as queuers (and we search for it by using `find_queuer_for` function).